src/lender/adapters/mkr/assessor.sol

```diff
@@ -89,7 +101,17 @@ contract MKRAssessor is Assessor {
     }
 
     function totalBalance() public view returns(uint) {
-        return safeAdd(reserve.totalBalance(), clerk.remainingCredit());
+        uint debt = clerk.debt();
+        // over the time the remainingCredit will decrease because of the accumulated debt interest
+        // therefore a buffer is reduced from the  remainingCredit to prevent the usage of currency which is not available
+        uint debtIncrease = safeSub(rmul(rpow(clerk.stabilityFee(),
+            safeSub(safeAdd(block.timestamp, creditBufferTime), block.timestamp), ONE), debt), debt);
```

Comment:
`safeSub(safeAdd(block.timestamp, creditBufferTime), block.timestamp) = creditBufferTime`

Reply:
True. 👍 
It happened because i wanted to use the interest lib first.